markdown/runtime/src/main/java/io/quarkiverse/qute/web/markdown/runtime/QuteMarkdownConfigurator.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkiverse.qute.web.markdown.runtime;
+
+import jakarta.enterprise.event.Observes;
+
+import io.quarkiverse.qute.web.markdown.runtime.commonmark.CommonMarkConverter;
+import io.quarkus.qute.EngineBuilder;
+
+public class QuteMarkdownConfigurator {
+
+    void addMarkdownSection(@Observes EngineBuilder builder) {
```

Comment:
Did you read my comment in https://github.com/quarkiverse/quarkus-qute-web/pull/92#discussion_r1654287077?

I prefer `@EngineConfiguration` because in that case the section helper can be used during the build.